src/Exceptions/UnauthorizedInvoiceAccessException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+namespace Laravel\Cashier\Exceptions;
+
+use Exception;
+use Throwable;
+
+class UnauthorizedInvoiceAccessException extends Exception
```

Comment:
Do we still support cases like admin access?

Reply:
True... I will remove it.